packages/docusaurus-theme-openapi/src/theme/ApiDemoPanel/ParamOptions/index.tsx

```diff
@@ -161,10 +166,16 @@ function ArrayItem({
 }
 
 function ParamArrayFormItem({ param }: ParamProps) {
-  const [items, setItems] = useState<{ id: string; value?: string }[]>([]);
+  const [items, setItems] = useState<Item[]>([]);
   const dispatch = useTypedDispatch();
 
   function handleAddItem() {
```

Comment:
Do we need to call updateItem here?

Reply:
Functionally I think it doesn't matter if we don't call it because at the point we add a new item it has no value yet, so we'd just pass over the value/strip it when we render the code examples. We only add it to the code example once it has a value (which is covered by the `handleChangeItem()` event).

An the other hand, the value of `items` has changed and calling `updateItems()` doesn't do any harm. So maybe from an "ideological" perspective, we should?

Again, I don't have a strong view on this one.